src/Components/Onboarding/Steps/Budget.tsx

```diff
@@ -1,70 +1,90 @@
 import * as React from "react"
+import { commitMutation, graphql } from "react-relay"
 import styled from "styled-components"
 
+import { ContextConsumer, ContextProps } from "../../Artsy"
 import SelectableToggle from "../SelectableToggle"
 import { StepProps } from "../Types"
 import { Layout } from "./Layout"
 
 const OptionsContainer = styled.div`
-width: 450px;
-margin: 0 auto 100px;
-&:last-child {
-  border-bottom: 1px solid #e5e5e5;
-}
+  width: 450px;
+  margin: 0 auto 100px;
+  &:last-child {
+    border-bottom: 1px solid #e5e5e5;
+  }
 `
 
 interface State {
-  selection: string
+  selection: number
```

Comment:
You set the initial state to `null`, so this should really be `number | null`.

Reply:
Ah yes, this makes sense! Good catch 👌